mido/midifiles/midifiles.py

```diff
@@ -38,7 +44,19 @@ def print_byte(byte, pos=0):
     print('  {:06x}: {:02x}  {}'.format(pos, byte, char))
 
 
-class DebugFileWrapper(object):
+class FileWrapper(object):
+    def __init__(self, file, error_mode='raise'):
```

Comment:
Ah, well spotted. Also, `FileWrapper` is never used. I don't know what I was thinking there. I'll remove it.